src/com/team2502/robot2018/Constants.java

```diff
@@ -34,87 +32,78 @@
+//    @Deprecated
+//    public static final float VR_MAX = 16F;
+//    @Deprecated
+//    public static final float VL_MAX = 16F;
+//    @Deprecated
+//    public static final float VR_MIN = -16F;
+//    @Deprecated
+//    public static final float VL_MIN = -16F;
 
+    // The max change in voltage per second (acceleration)
+//    public static final float AR_MAX = 4F;
+//    public static final float AL_MAX = 4F;
+//    public static final float AR_MIN = -4F;
+//    public static final float AL_MIN = -4F;
     // The lookahead distance (feet) for Pure Pursuit
-    public static final float LOOKAHEAD_MIN_DISTANCE_FT = 3F;
-    public static final float LOOKAHEAD_MAX_DISTANCE_FT = 5F;
+    public static final float LOOKAHEAD_MIN_DISTANCE_FT = 2F;
+    public static final float LOOKAHEAD_MAX_DISTANCE_FT = 5.5F;
     public static final float LOOKAHEAD_MIN_SPEED_FPS = 1F;
-    public static final float LOOKAHEAD_MAX_SPEED_FPS = 8F;
+    public static final float LOOKAHEAD_MAX_SPEED_FPS = 10F;
     public static final Lookahead LOOKAHEAD = new Lookahead(LOOKAHEAD_MIN_DISTANCE_FT, LOOKAHEAD_MAX_DISTANCE_FT,
                                                             LOOKAHEAD_MIN_SPEED_FPS, LOOKAHEAD_MAX_SPEED_FPS);
-    public static final float STOP_DIST_TOLERANCE_FT = 1F;
+    public static final float STOP_DIST_TOLERANCE_FT = 2F;
+    // TODO: figure out why wheel diameter has to be much smaller than it should be (normally 6)
+    public static final float WHEEL_DIAMETER_INCH = 6F; // 3.6944444443F;
+
+    public static final float WHEEL_REV_TO_ENC_REV_LOW = 4.285F;
+    public static final float WHEEL_REV_TO_ENC_REV_HIGH = 2.083F;
 
     /*
     Physical / Other
      */
-
-
-    // TODO: figure out why wheel diameter has to be much smaller than it should be (normally 6)
-    public static final float WHEEL_DIAMETER_INCH = 3.6944444443F;
     public static final float WHEEL_DIAMETER_FT = WHEEL_DIAMETER_INCH / 12F;
-
     // half of the width of the wheel that is in contact with the ground
     public static final float WHEEL_ROLLING_RADIUS_INCH = 1.5F;
     public static final float WHEEL_ROLLING_RADIUS_FT = WHEEL_ROLLING_RADIUS_INCH * 1.5F / 12F;
-
     // The distance between wheels (laterally) in feet. Measure from the centerpoints of the wheels.
     public static final float LATERAL_WHEEL_DISTANCE_FT = 23.25F / 12F * 5F; //* 10F;
-
     public static final float MAX_FPS_SPEED = 18.0F;
-
     public static final float ENC_RES = 4096.0F;
-
-
     /* Drivetrain */
     public static final float EPOS_TO_FEET_DT = (WHEEL_DIAMETER_FT * (float) Math.PI) / ENC_RES;
-
     // 600 = amount of 100 ms in a minute
     public static final float EVEL_TO_RPM = (600.0F / ENC_RES);
     public static final float RPM_TO_FPS_DT = (WHEEL_DIAMETER_FT * (float) Math.PI) / 60F;
-    public static final float EVEL_TO_FPS_DT = EVEL_TO_RPM * RPM_TO_FPS_DT;
 
-    public static final float FEET_TO_EPOS_DT = 1 / EPOS_TO_FEET_DT;
+    //TODO: change bad name
```

Comment:
Preferably, we should change this name before merging into the pristine master branch